__mocks__/@react-native-firebase/crashlytics.ts

```diff
@@ -0,0 +1,19 @@
+import type {FirebaseCrashlyticsTypes} from '@react-native-firebase/crashlytics';
+
+type CrashlyticsModule = {
+    log: FirebaseCrashlyticsTypes.Module['log'];
+    recordError: FirebaseCrashlyticsTypes.Module['recordError'];
+    setCrashlyticsCollectionEnabled: FirebaseCrashlyticsTypes.Module['setCrashlyticsCollectionEnabled'];
+};
```

Comment:
```suggestion
type CrashlyticsModule = Pick<FirebaseCrashlyticsTypes.Module, 'log' | 'recordError' | 'setCrashlyticsCollectionEnabled'>;
```

Reply:
Thank you for the suggestion. Make sense!